kinto/config/__init__.py

```diff
@@ -9,7 +9,11 @@ def render_template(template, destination, **kwargs):
     template = os.path.join(HERE, template)
 
     folder = os.path.dirname(destination)
-    os.makedirs(folder)
+    if os.path.exists(folder):
+            print("%s already exists. kinto.ini will be created" % folder)
```

Comment:
We probably don't want to print out here. Just let the "if not os.path.exists(folder)", and add a test :-)